### DISCUSSION TITLE
[Request] Add Vista's Aero effect to DWMBG

### DISCUSSION THREAD

**Discussion by TheInsane101** [Maplespe/DWMBlurGlass#337] [Q&A]
Vruh explained how in 2.1, the non-Aero "Blur" option in the list of blurs you can choose under the "CustomBlur" option was changed from being a generic Gaussian blur with a single colour layer and nothing else, to having the multiply blur effect that the Windows 7 Aero has, except that it doesn't have the three ColorizationBalance effects of 7. 

While it's nice to have the multiply blend blur, it would be nice to also have the option to use Vista's blur, which is the generic Gaussian blur with no extra effects and one basic colour layer whose opacity is adjusted by the Intensity slider. I've put together all the colour hex codes and opacity values and subsequent config files for all Vista default colours as well as the Vista beta 2 default colours, which are different, and these configs are only useful for achieving an accurate Vista look with the original blur.

[Vruh currently has made a fork that replaces the multiply blend blur with the Vista blur,](https://github.com/Maplespe/DWMBlurGlass/discussions/309#discussioncomment-9314755), but it would be nice to have this back in the original DWMBG, either as an additional option or replacing the current Blur.

**Comment by InsaneGal** [comment]
> I've put together all the colour hex codes and opacity values and subsequent config files for all Vista default colours as well as the Vista beta 2 default colours

Can you share these configs pls?

**Reply by TheInsane101** [reply]
It's literally the discussion thread third from the top https://github.com/Maplespe/DWMBlurGlass/discussions/309
